.github/workflows/tests-rs-package.yml

```diff
@@ -13,6 +13,10 @@ on:
         description: Runner for linting. Must be JSON valid string.
         type: string
         default: '"ubuntu-22.04"'
+      check-features:
```

Comment:
check-each-feature

Reply:
We testing all features already and it's not very understandable that with this flag is we will check them independently